### DISCUSSION TITLE
Directly access SST files generated by other databases

### DISCUSSION THREAD

**Discussion by wenhaocs** [ClickHouse/ClickHouse#47661] [General]
Hi,

In our environment, we have OLTP database uses rocksdb as the backend. We are wondering if there is a way to use ClickHouse to query data from those SSTs directly. I am not sure whether the `EmbeddedRocksDB` engine can work with SSTs generated by other databases. 

Thanks!

**Comment by mdonkers** [comment]
I'm afraid currently this isn't possible. The list of formats that ClickHouse supports (https://clickhouse.com/docs/en/interfaces/formats) does not list the BlockBasedTable (https://github.com/facebook/rocksdb/wiki/Rocksdb-BlockBasedTable-Format) from RocksDB.

I think you'll need some other way to sync databases.
Not sure on the possibilities of RocksDB, but e.g. ClickHouse supports creating Materialized Views that export data to some other format / location. Perhaps RocksDB supports something similar and you can let ClickHouse read data from there?
